frontend/packages/dev-console/src/components/topology/topology-utils.ts

```diff
@@ -0,0 +1,436 @@
+import * as _ from 'lodash-es';
+import { LabelSelector } from '@console/internal/module/k8s/label-selector';
+import { getRouteWebURL } from '@console/internal/components/routes';
+import { TopologyDataResources, ResourceProps, TopologyDataModel } from './topology-types';
+
+export const podColor = {
+  Running: '#0066CC',
+  'Not Ready': '#519DE9',
+  Warning: '#F0AB00',
+  Empty: '#FFFFFF',
+  Failed: '#CC0000',
+  Pending: '#8BC1F7',
+  Succceeded: '#519149',
+  Terminating: '#002F5D',
+  Unknown: '#A18FFF',
+  'Scaled To 0': '#FFFFFF',
+};
+
+export const podStatus = Object.keys(podColor);
+
+function numContainersReadyFilter(pod) {
+  let numReady = 0;
+  _.forEach(pod.status.containerStatuses, function(status) {
```

Comment:
style nit: prefer `=>`

(I'm guessing a lot of this logic came from origin-web-console)

Reply:
I can do a sweep now. Anytime we have nits, we should attempt to fix them with eslint so we do not have to keep coming back to them.